tests/test_lwscs.py

```diff
@@ -0,0 +1,416 @@
+
+import logging
+import math
+import unittest
+
+import pytest
+from lsst.ts import mtdome
+from lsst.ts.mtdome.mock_llc.lwscs import NUM_MOTORS, POWER_PER_MOTOR
+
+logging.basicConfig(
+    format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.DEBUG
+)
+
+_start_tai = 10001.0
+_MIN_POSITION = 0
+_MAX_POSITION = math.radians(90)
+
+
+class LwscsTestCase(unittest.IsolatedAsyncioTestCase):
+    async def prepare_lwscs(
+        self,
+        start_position: float,
+        min_position: float,
+        max_position: float,
+        max_speed: float,
+        start_tai: float,
+    ) -> None:
+        """Prepare the LWSCS for future commands.
+
+        Parameters
+        ----------
+        start_position: `float`
+            The initial position.
+        min_position: `float`
+            The minimum allowed position.
+        max_position: `float`
+            The maximum allowed position.
+        max_speed: `float`
+            The maximum allowed speed.
+        start_tai: `float`
+            The current TAI time.
+        """
+        self.lwscs = mtdome.mock_llc.LwscsStatus(start_tai=start_tai)
+        elevation_motion = mtdome.mock_llc.ElevationMotion(
+            start_position=start_position,
+            min_position=min_position,
+            max_position=max_position,
+            max_speed=max_speed,
+            start_tai=start_tai,
+        )
+        self.lwscs.elevation_motion = elevation_motion
+
+    async def verify_lwscs_motion(
+        self,
+        tai: float,
+        expected_position: float,
+        expected_velocity: float,
+        expected_motion_state: mtdome.LlcMotionState,
+    ) -> None:
+        """Verify the position of the LWSCS at the given TAI
+        time.
+
+        Parameters
+        ----------
+        tai: `float`
+            The TAI time to compute the position for.
+        expected_position: `float`
+            The expected position at the given TAI time.
+        expected_velocity: `float`
+            The expected velocity at the given TAI time.
+        expected_motion_state: `float`
+            The expected motion state at the given TAI time.
+        """
+        await self.lwscs.determine_status(current_tai=tai)
+        assert expected_position == pytest.approx(
+            self.lwscs.llc_status["positionActual"]
+        )
+        assert expected_velocity == pytest.approx(
+            self.lwscs.llc_status["velocityActual"]
+        )
+        assert expected_motion_state.name == self.lwscs.llc_status["status"]["status"]
+        expected_drive_current: list[float] = [0.0] * NUM_MOTORS
+        if expected_motion_state in [
+            mtdome.LlcMotionState.CRAWLING,
+            mtdome.LlcMotionState.MOVING,
+        ]:
+            expected_drive_current = [POWER_PER_MOTOR] * NUM_MOTORS
+        assert expected_drive_current == self.lwscs.llc_status["driveCurrentActual"]
+
+    async def test_move_zero_ten(self) -> None:
```

Comment:
Another case for a `check_move(...)` method (though you have done a lot of that already with verify_lwscs_motion, which is great).